test/DynamoCoreWpfTests/DateTimePaletteTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using CoreNodeModels.Input;
+using Dynamo;
+using Dynamo.Configuration;
+using Dynamo.Tests;
+using Dynamo.Graph.Workspaces;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    class DateTimePaletteTests : DynamoModelTestBase
```

Comment:
could these tests be moved to DynamoCore test - is there anything here that is actually testing the wpf components or building a view?